src/main/java/no/nav/familie/integrasjoner/client/rest/SafRestClient.kt

```diff
@@ -28,46 +33,51 @@ class SafRestClient(@Value("\${SAF_URL}") safBaseUrl: URI,
     fun hentJournalpost(journalpostId: String): Journalpost {
         val safJournalpostRequest = SafJournalpostRequest(SafRequestVariabler(journalpostId),
                                                           graphqlQuery("/saf/journalpostForId.graphql"))
-        try {
-            val response = postForEntity<SafJournalpostResponse<SafJournalpostData>>(safUri,
-                                                                                     safJournalpostRequest,
-                                                                                     httpHeaders())
-            if (response != null && !response.harFeil()) {
-                return response.data?.journalpost ?: throw JournalpostRestClientException("Kan ikke hente journalpost",
-                                                                                          null,
-                                                                                          journalpostId)
+        val response = postForEntity<SafJournalpostResponse<SafJournalpostData>>(safUri,
+                                                                                 safJournalpostRequest,
+                                                                                 httpHeaders())
+        if (!response.harFeil()) {
+            return response.data?.journalpost ?: throw JournalpostRestClientException("Kan ikke hente journalpost",
+                                                                                      null,
+                                                                                      journalpostId)
+        } else {
+            val tilgangFeil = response.errors?.firstOrNull { it.message?.contains("Tilgang til ressurs ble avvist") == true }
+
+            if (tilgangFeil != null) {
+                throw JournalpostForbiddenException(tilgangFeil.message)
             } else {
                 responsFailure.increment()
```

Comment:
Burde denne flyttet opp, sånn att den blir +1 for alle typer feil?

Reply:
Tilgangskontroll er ikke feil tenker jeg.